<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qlnvWindow</class>
 <widget class="QMainWindow" name="qlnvWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgba(210, 210, 210,1);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tb_qlnv">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>540</width>
      <height>291</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tên</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Địa chỉ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SDT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Chức vụ</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Lương/tháng</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="lb_qlnv">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Quản lý nhân viên</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="tim_qlnv">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>50</y>
      <width>411</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
    </property>
    <property name="placeholderText">
     <string>Tìm kiếm</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grb_qlch">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>270</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(210, 210, 210,1);</string>
    </property>
    <property name="title">
     <string>Thông tin</string>
    </property>
    <widget class="QLineEdit" name="txt_qlnv_tn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
     </property>
     <property name="placeholderText">
      <string>Tên</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_qlnv_dc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
     </property>
     <property name="placeholderText">
      <string>Địa chỉ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_qlnv_sdt">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
     </property>
     <property name="placeholderText">
      <string>Số điện thoại</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_qlnv_cv">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
     </property>
     <property name="placeholderText">
      <string>Chức vụ</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_qlnv_ten">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tên</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_qlnv_dc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>106</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Địa chỉ</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_qlnv_sdt">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SDT</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_qlnv_cv">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Chức vụ</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_qlnv_l">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Lương</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_qlnv_l">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>230</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(255, 255, 255, 1);</string>
     </property>
     <property name="placeholderText">
      <string>Lương</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_timkiem_qlnv">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(118, 203, 248, 0.8);
</string>
    </property>
    <property name="text">
     <string>Tìm kiếm</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btn_again">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(118, 203, 248, 0.8);
</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sua_qlnv">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(118, 203, 248, 0.8);
</string>
     </property>
     <property name="text">
      <string>Sửa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_thoat_qlnv">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(225, 130, 146, 1);</string>
     </property>
     <property name="text">
      <string>Thoát</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_them_qlnv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(118, 203, 248, 0.8);
</string>
     </property>
     <property name="text">
      <string>Thêm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_xoa_qlnv">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(215, 220, 88, 0.77);</string>
     </property>
     <property name="text">
      <string>Xoá</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
